.expeditor/config.yml

```diff
@@ -0,0 +1,12 @@
+---
+project:
+  alias: homebrew-habitat
+
+github:
+  delete_branch_on_merge: true
+
+subscriptions:
+  - workload: project_promoted:{{agent_id}}:stable:*
```

Comment:
Won't we need to replace `{{agent_id}}` with `habitat-sh/habitat:master`?

Reply:
I also think this would have to be `current`, rather than `stable`, since it needs to be the source of the thing being promoted, and not the destination.

See https://github.com/habitat-sh/habitat/blob/5e4056165b73f226f8fcece183521f68828dc424/.expeditor/config.yml#L114-L116

I'm also wondering about race conditions now, because I can imagine the Habitat repo's responses and this repo's responses both being kicked off around the same time, which could result in this script figuring out what's in `stable` before the other repo's script does the actual move. We'd end up with a PR for the last stable thing, rather than the new stable thing.

I _think_ we want to subscribe to 
```
habitat-sh/habitat:master_completed:project_promoted:habitat-sh/habitat:master:current:*
```
based on https://expeditor.chef.io/docs/reference/workloads/#completed-workloads, or possibly `habitat_completed:project_promoted:habitat:current:*`, if the `project.alias` can be used in this situation, and not just with Slack promotion commands.

Help us @tduffield ; you're our only hope!

Reply:
Yeah, you probably want to subscribe to the completed workload, but that stuff doesn't respect project aliases.